sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -540,7 +537,12 @@ def send_message(
             process_storage_error(error)
 
     @distributed_trace
-    def receive_message(self, **kwargs: Any) -> Optional[QueueMessage]:
+    def receive_message(
+            self,
+            *,
+            visibility_timeout: Optional[int] = None,
+            **kwargs: Any
+        ) -> QueueMessage:
```

Comment:
Missing `Optional` on new return type. Probably missed because I just recently made this change in `main` :)